### DISCUSSION TITLE
Card view problem master detail relation ship

### DISCUSSION THREAD

**Discussion by NikolaS4yourb** [serenity-is/Serenity#6769] [Help]
Card mix view relationship problem

Hi,
I use card view mixin . When I master file grid  put on 'card-view' ,  and  click on Edit,  master detail relationship on child do not show data.

 
![image](https://github.com/serenity-is/Serenity/assets/88085419/3c83ebe8-b280-4973-b2f9-75c6565d1a07)

 
![image](https://github.com/serenity-is/Serenity/assets/88085419/cdd87d4b-ef32-406f-8961-cf2548ba8e18)

When I back and change to list view, data is shown.

 
![image](https://github.com/serenity-is/Serenity/assets/88085419/393e7776-9402-4931-92e7-08369d4c8ab1)

**Comment by VictorTomaili** [comment]
Card view mixin is a pro feature. You can ask in StartSharp repository instead. Please consider the license agreement and avoid sharing the source code of card view mixin.
